src/mobile/apps/show/test/helpers/view_helpers.test.coffee

```diff
@@ -0,0 +1,33 @@
+sinon = require 'sinon'
```

Comment:
Meta-comment that I missed before (so feel free to ignore!) but this file (and `view_helpers` above) can be written in ES6 syntax vs. coffeescript, as that's what we prefer.

No problem if you don't want to make this change this time around!

Reply:
😥